README.md

```diff
@@ -37,6 +37,10 @@ I think this is kind of silly, but that's just me.
 
 ## Usage
 
+This action runs in 2 variants. By default it'll create a unique DOI every single time.
+However, If you specify `doi` that represents all versions, it will create *new versions* of that existing
+DOI.  These are essentially new DOIs, but they're associated in Zenodo with the `doi` for *all* versions.
```

Comment:
Can't say that this is the best spot for this, but it's the spot I used.

Reply:
Oh that's perfect! Would you mind including that screen shot in the README (add an img/ folder for it) so it's clear? And it looks like my previous upload (associated with the token) has one too! So if you want we can use that for the CI test, perhaps with a staging URL? [10.5281/zenodo.6326822](https://doi.org/10.5281/zenodo.6326822)

Reply:
I can do the docs, but I don't know about url toggles. Can that be a separate PR?

Reply:
Sure! What I can do is make a PR that starts with your branch and then adds it, and triggers a test. Just ping me when you are done and I can make some time in an evening.

Reply:
I think docs are OK and I just added alt text, so I believe I'm done.

Reply:
Awesome thanks! I'll do some more digging tonight. Thanks for your hard work on this today! :partying_face: